lookout/style/typos/corrector.py

```diff
@@ -75,6 +80,7 @@ def initialize_ranker(self, boost_params: Optional[dict] = None,
         """
         boost_params = boost_params or self.DEFAULT_BOOST_PARAM
         self.ranker.construct(boost_params, train_rounds, early_stopping)
+        self._log.debug("Ranker %s is initialized.", repr(self.ranker))
```

Comment:
If `repr` includes the class name, we should remove redundant "Ranker" from the logs.